server/appServices.js

```diff
@@ -138,6 +144,22 @@ async function watchKeyCloakSecrets(namespace, kubeclient) {
   secretStream.on('end', () => {
     watchKeyCloakSecrets(namespace, kubeclient);
```

Comment:
Why keeping this?

Reply:
`end` is different. It is when the watch on the resource is closed. `close` happens when the kubernetes API is restarted.

```

--min-request-timeout int     Default: 1800
An optional field indicating the minimum number of seconds a handler must  keep a request open before timing it out. Currently only honored by the  watch request handler, which picks a randomized value above this number  as the connection timeout, to spread out load.

```

Reply:
Are you sure this doesn't result in a `close` event too?

```
The 'end' event is emitted when there is no more data to be consumed from the stream.
```